src/gm3/components/serviceManager.js

```diff
@@ -82,6 +82,18 @@ function getExtentForQuery(results) {
             }
         }
     }
+
+    if (extent[3] - extent[1] < minSize || extent[2] - extent[0] < minSize) {
```

Comment:
This should probably be separate conditions for x and y.  If the real extent is 1 x 1000 (say a street segment) this will shrink it to 150x150.  (Or... maybe it should be an `&&` instead of a `||`)